### ISSUE TITLE
add an option to specify multiple streamers

### ISSUE THREAD

**Issue by Andrey1994** [brainflow-dev/brainflow#509]
**Use case:**
stream data to different destinations and after #494 stream different data types to different destinations

In terms of code it more likely should look like:
```
board->add_streamer();
```
And probably it should be removed from `start_stream` arguments

*It more likely will be a breaking change so it should be postponed until there is a necessity in other breaking changes*
